src/aura/CRLP_FilterGroup/CRLP_FilterGroupController.js

```diff
@@ -228,13 +238,13 @@
                 //set field labels directly
                 filterRule.objectLabel = helper.retrieveFieldLabel(filterRule.objectName, cmp.get("v.detailObjects"));
                 filterRule.fieldLabel = helper.retrieveFieldLabel(filterRule.fieldName, cmp.get("v.filteredFields"));
-                filterRule.operatorLabel = helper.retrieveFieldLabel(filterRule.operatorName, cmp.get("v.filteredOperators"));
+                filterRule.operationLabel = helper.retrieveFieldLabel(filterRule.operation, cmp.get("v.filteredOperators"));
 
                 //special reformatting for multipicklist and semi-colon delimited lists
-                if (filterRule.operatorName === 'In_List' || filterRule.operatorName === 'Not_In_List') {
-                    filterRule.constantLabel = helper.reformatConstantLabel(cmp, filterRule.constantName, filterRule.operatorName);
+                if (filterRule.operation === 'In_List' || filterRule.operation === 'Not_In_List') {
+                    filterRule.valueLabel = helper.reformatvalueLabel(cmp, filterRule.value, filterRule.operation);
```

Comment:
Should use camel case for reformatvalueLabel function

Reply:
Great catch. I didn't mean to rename the function from `reformatConstantLabel` to `reformatValueLabel` so this was actually failing. It's fixed now.